geometry-1/DOMPoint-002.html

```diff
@@ -0,0 +1,171 @@
+        test(function() {
+            checkDOMPoint(new DOMPointReadOnly(1, 2, 3, 4), {x:1, y:2, z:3, w:4});
+        },'test DOMPointReadOnly Constructor4');
+        test(function() {
+            checkDOMPoint(new DOMPointReadOnly(1, 2, 3, 4, 5), {x:1, y:2, z:3, w:4});
+        },'test DOMPointReadOnly Constructor5');
+        test(function() {
+            checkDOMPoint(new DOMPointReadOnly(1, undefined), {x:1, y:0, z:0, w:1});
+        },'test DOMPointReadOnly Constructor6 undefined');
+        test(function() {
+            checkDOMPoint(new DOMPointReadOnly("a", "b"), {x:NaN, y:NaN, z:0, w:1});
+        },'test DOMPointReadOnly Constructor7 undefined');
+        test(function() {
+            checkDOMPoint(new DOMPointReadOnly({}), {x:NaN, y:0, z:0, w:1});
+        },'test DOMPointReadOnly Constructor8 Undefined');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({}), {x:0, y:0, z:0, w:1});
+        },'test DOMPointReadOnly fromPoint0');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({x:1}), {x:1, y:0, z:0, w:1});
+        },'test DOMPointReadOnly fromPoint1');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({x:1, y:2}), {x:1, y:2, z:0, w:1});
+        },'test DOMPointReadOnly fromPoint2');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({x:1, y:2, z:3}), {x:1, y:2, z:3, w:1});
+        },'test DOMPointReadOnly fromPoint3');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({x:1, y:2, z:3, w:4}), {x:1, y:2, z:3, w:4});
+        },'test DOMPointReadOnly fromPoint4');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({x:1, y:2, z:3, w:4, v:5}), {x:1, y:2, z:3, w:4});
+        },'test DOMPointReadOnly fromPoint5');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({x:1, z:3}), {x:1, y:0, z:3, w:1});
+        },'test DOMPointReadOnly fromPoint6 irregular');
+        test(function() {
+            checkDOMPoint(DOMPointReadOnly.fromPoint({x:1, y: undefined, z:3}), {x:1, y:0, z:3, w:1});
+        },'test DOMPointReadOnly fromPoint7 undefined');
+        test(function() {
+            var point = new DOMPointReadOnly(5, 4);
+            var matrix = new DOMMatrix([2, 0, 0, 2, 10, 10]);
+            var result = point.matrixTransform(matrix);
+            var expected = getMatrixTransform(matrix, point);
+            checkDOMPoint(result, expected);
+        },'test DOMPointReadOnly matrixTransform');
+        test(function() {
+            var p = new DOMPointReadOnly(0, 0, 0, 1);
+            p.x = undefined;
+            p.y = undefined;
+            p.z = undefined;
+            p.w = undefined;
+            checkDOMPoint(p, {x:0, y:0, z:0, w:1});
+        },'test DOMPointReadOnly Attributes');
+
+        function checkDOMPoint(p, exp) {
+            assert_equals(p.x, exp.x, "Expected value for x is " + exp.x);
```

Comment:
The generated message for `assert_equals` already says what the expected value is. Just need to say which property is being asserted.